badparking/core/models.py

```diff
@@ -11,3 +13,23 @@ def __unicode__(self):
     class Meta:
         verbose_name = "Тип порушення"
         verbose_name_plural = "Типи порушень"
+
+
+class Claim(models.Model):
+    license_plates = models.CharField("Номери автомобілів", max_length=50, blank=True)
+    longitude = models.CharField("Довгота", max_length=50, blank=True)
+    latitude = models.CharField("Широта", max_length=50, blank=True)
+    city = models.CharField("Місто", max_length=30, blank=True)
```

Comment:
I'd recommend not limiting `max_length` so strictly on fields like `city` and `address` as you never know... 255 is a good default choice.